source/geometry/Quaternion.ooc

```diff
@@ -146,8 +138,7 @@ Quaternion: cover {
 	distance: func (other: This) -> Float { (this - other) norm }
 	angle: func (other: This) -> Float {
 		result := 2.0f * (this dotProduct(other) absolute) acos()
-		result = result == result ? result : 0.0f
-		result
+		result == result ? result : 0.0f
```

Comment:
It works if I add `isNumber ::= this == this` to `Float: cover...`. Must be messing up in the implicit cast or something.